components/Question.tsx

```diff
@@ -9,7 +10,7 @@ interface QuestionProps {
   answer: string;
   htmlFor: string;
   name: string;
-  options: Option[];
+  options: OptionProps[];
```

Comment:
```suggestion
  options: Option[];
```

Reply:
Option is a component from antd im gunna be using once i figure out how to get past that error

Reply:
might as well rename it now

Reply:
you can alias the import `import { Option as OptionComponent } from 'blah';`

Reply:
but isnt Option a prop? you wrote a similar property. just like you did  `QuestionProps`

Reply:
that naming convention is referring to the interface of props being passed into the component. for interfaces, the naming convention is `<name_of_component>Props`. So if the component was called `RadioButton` the props interface would be `RadioButtonProps`. Since `option` is inside of `QuestionProps` the type of `option` wouldn't be `OptionProps` with our naming convention.